### DISCUSSION TITLE
Append to Daily Note

### DISCUSSION THREAD

**Discussion by teqdeq** [coddingtonbear/obsidian-web#95] [Exotic Browser Support]
Hello,

I am trying Append to a Daily Note, but can't figure out how to set it up.
My Daily Notes are under vault/_Journal/Daily

Creating new Notes is no problem, but how do i get the todays daily note Url?

Thx
Jan

**Comment by ghost** [comment]
Hello,

I've been trying to figure this out myself and looks like this works:

`/vault/inbox/{{date "yyyy-MM-dd"}}.md`

1. `vault` has to be named `vault` AFAIK. Don't name it `potato` even if that's your Vault's name.
2. The `MM` have to be capitalized, otherwise they'll mean _minutes_.
3. The example given in the default template is IMHO misleading. I installed the Extension on a fresh Brave profile just to be sure and the API points to `/periodic/daily/` which never worked for me. @coddingtonbear Can I request that you update the default API path? And maybe add a small bit of inline documentation? Thanks for the dope project! ⭐

**Comment by coddingtonbear** [comment]
1. Yes, of course it must remain `/vault/` -- that's part of the path to the API for retrieving a file from your _vault_.   At the moment it's not at all clear to me how the documentation could be interpreted in another way, though, so I have to say I'm curious to see how you were interpreting this: https://coddingtonbear.github.io/obsidian-local-rest-api/.
2. The date formatting operators that are available are documented here: https://github.com/coddingtonbear/obsidian-web/wiki/Understanding-Templates.  You'll note that those are not something I'm inventing -- they come from a library we're relying upon.
3. I'm afraid `/periodic/daily/` absolutely does work.  I use it myself every day.  You do need to be using the periodic notes obsidian plugin, of course, though.  That being said, I cannot personally provide support for anything other than Chrome, and nobody is going to be able to help point you in the right direction at all unless you provide some kind of indication of _how_ things are going wrong.

**Reply by ricir** [reply]
It seems to me that the Periodic Notes plugin is not mandatory for Obsidian Web to work with the Daily Note. I've been using it for a long time and it works perfectly. I think Local REST API has some kind of fallback to convert `/periodic/daily/` to the actual path defined in the core Daily Notes plugin—or something along those lines.

Anyway, I don't have the Periodic Notes plugin installed, but with the latest version of Local REST API, I can use the `/periodic/daily/` API URL to append content to my current daily note. I did notice that a recent release of Local REST API broke this feature, but the latest update has restored it

**Comment by ghost** [comment]
1. I admit, I skimmed this part. I relied merely on the hint in the form field. My bad!
2. That was not an accusation on my part. I was merely helping OP.
3. I still recommend changing the provided example, since a new Obsidian user such as myself (an maybe OP) may not even know that this plugin exists. I learned this just now from you. I'm also aware of Brave not being explicitly supported and I'm OK with that.

Either way, thanks for the extension!

**Comment by coddingtonbear** [comment]
Oh, sorry @tangodev, I absolutely didn't notice that you were replying to somebody else's question (both you and @teqdeq have generic user icons and a username starting with `t`!) and definitely thought you were posing three bullet points of questions.  Thanks for trying to help somebody out, and sorry for misunderstanding!

Re (3): I'm afraid the only choice I'd have for following through with your suggestion is just removing that template entirely and I'm not convinced that that's on average going to make peoples' lives less confusing.  As I'm sure you're aware, everything around the built-in daily notes feature is customizable so there are no consistent file locations or naming conventions that everybody uses.  Without that, there's no way for me to make any template at all for daily notes that could be reliable.  The periodic notes plugin, though, _does_ make that reliable by providing an API obsidian-local-rest-api can interact with for figuring out where the relevant note should be created/retrieved and what its name should be/is.  I suppose we should probably note that this relies on the obsidian-periodic-notes plugin in the docs here, though: https://coddingtonbear.github.io/obsidian-local-rest-api/#/Periodic%20Notes/get_periodic__period__ (for each of the endpoints under `/periodic/`) and would totally accept a PR adding that to the docs.
